README.md

```diff
@@ -19,53 +19,161 @@ how it works in [this blog post](http://blog.humphd.org/thimble-and-bramble/).
+- g++ 4.8 or above (for the login.webmaker.org dependency)
+- [Webmaker Login Server](#loginwebmakerorg)
+
+The following is an abbreviated guide to getting it all set up. Please see each server's README for more details.
+
+### Manually Installing the Parts
+Please note: On Windows, use ``copy`` instead of ``cp``
+
+####Thimble
+* Fork and clone https://github.com/mozilla/thimble.mozilla.org
+* Run ``cp env.dist .env`` to create an environment file
+* Run ``npm install`` to install dependencies
+* Run ``npm start`` to start the server
+* Once everything is ready and running, Thimble will be available at [http://localhost:3500/](http://localhost:3500/)
+
+####id.webmaker.org
+* Clone https://github.com/mozilla/id.webmaker.org
+* Run ``cp sample.env .env`` to create an environment file
+* Run ``npm install`` to install dependencies
+* Run ``npm start`` to start the server
+
+####login.webmaker.org
+* Clone https://github.com/mozilla/login.webmaker.org
+* Run ``npm install`` to install dependencies
+* Run ``cp env.sample .env`` to create an environment file
+* Run ``npm start`` the server
+
+####PostgreSQL
+* Run ``initdb -D /usr/local/var/postgres`` to initialize PostreSQL
+  * If this already exists, run ``rm -rf /usr/local/var/postgres`` to remove it
+* Run ``postgres -D /usr/local/var/postgres`` to start the PostgreSQL server
+* Run ``createdb publish`` to create the Publish database
+
+####publish.webmaker.org
+* These steps assume you've followed the PostgreSQL steps above, including creating the publish database.
+* Clone https://github.com/mozilla/publish.webmaker.org
+* Run ``npm install`` to install dependencies
+* Run ``npm run env``
+* Run ``npm install knex -g`` to install knex
+* Run ``npm run knex`` to seed the publish database created earlier
+* Run ``npm start`` to run the server
+
+### Getting Ready to Publish
+To publish locally, you'll need to do the following...
+
+####Teach the ID server about the Publish server
+
+* Run ``createdb webmaker_oauth_test`` to create a test database
+* In your id.webmaker.org folder
+  * Run ``node scripts/create-tables.js``
+  * Edit ``scripts/test-data.sql`` and replace it's contents with:
+
+      ```sql
+        INSERT INTO clients VALUES
+          ( 'test',
+            'test',
+            '["password", "authorization_code"]'::jsonb,
+            '["code", "token"]'::jsonb,
+            'http://localhost:3500/callback' )
+      ```
+
+  * Run ``node scripts/test-data.js``
+    * You'll see a ``INSERT 0 1`` message if successful
+
+####Sign In
+
+To publish locally, you'll need an account.
+* Go to [http://localhost:3000/account](http://localhost:3000/account)
+* Click ``Join Webmaker`` and complete the process, you can use a fake email
+* When you've created your account, click ``Set permanent password instead``
+  * This lets you authenticate your account without needing email
+* Go back to Thimble and Log In with your new account
+
+# 2) Running the parts
```

Comment:
I would delete this whole section because it seems redundant once you make the changes I suggest above